packages/eth-sdk/src/config/types.ts

```diff
@@ -83,6 +89,7 @@ const ethSdkConfigSchema = z
     noFollowProxies: z.boolean().optional(),
     abiSource: abiSourceSchema.default(DEFAULT_ABI_SOURCE),
     networkIds: networkIdsSchema.default({}),
+    flags: flagsSchema.optional(),
```

Comment:
can we name this `typeChainFlags` instead of simply `flags`? I think it is confusing now.

Reply:
done!